CMakeLists.txt

```diff
@@ -1,20 +1,29 @@
-cmake_minimum_required(VERSION 3.12)
-project(Amethyst-Template CXX ASM_NASM)
-set(MOD_VERSION "0.0.0")
-set(MOD_AUTHOR "FrederoxDev")
+####################################################################################################
+set(SHARD_NAME "Shard Template")
+set(SHARD_AUTHOR "Author")
+set(SHARD_UUID "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX")
+set(SHARD_VERSION "0.0.0")
+
+# ADVANCED
+set(SHARD_FORMAT "MOD")
+set(SHARD_FORMAT_VERSION "3.0.0") # Don't modify
```

Comment:
Shard is just the file extension, continue referring to these as mods

Reply:
they aren't always mods though, and calling em shards aint gonna make much of a difference

Reply:
> they aren't always mods though, and calling em shards aint gonna make much of a difference 

bro who are you mad at 😂

Reply:
> > they aren't always mods though, and calling em shards aint gonna make much of a difference
> 
> bro who are you mad at 😂

no one? i'm just replying

Reply:
> they aren't always mods though, and calling em shards aint gonna make much of a difference 

Please stop giving so much resistance to everything, it's pointless. We are not renaming the word mod

Reply:
I'm not being resistant for no reason. Keeping the name as "Mod" is confusing;
- why have the format field? A mod "mod"? a mod "runtime"?
- why is the file name .shard but they're called mods?

Plus changing the variable names literally ain't gonna do anything. You can still can em mods if you want, but the actual naming has to be clear what it is. It's not specifically a mod.